docs/team/skills.md

```diff
@@ -0,0 +1,103 @@
+- **YAML Configuration**. JupyterHub uses YAML, a way of structuring text as configuration
+  files, in order to control the behavior of many tools in the JupyterHub ecosystem (as
+  well as to configure CI/CD infrastructure). [Here is a quick primer on YAML](https://github.com/darvid/trine/wiki/YAML-Primer).
+- **Testing infrastructure**. JupyterHub projects generally use [pytest](https://docs.pytest.org/en/stable/)
+  for testing. However, some projects involve more complex tests than you'd normally find
+  in a Python project. For example, mocking HTTP servers, or infrastructure that runs in the cloud.
+
+### Server infrastructure
+
+- **HTTP and REST APIs**. These are the ways in which services, extensions, and some
+  authenticators interact with a JupyterHub.
+- **tornado / asyncio**. JupyterHub uses these tools in order to send messages across
+  JupyterHub infrastructure (e.g., between JupyterHub processes and user servers).
+
+### UI/UX
+
+- **HTML, CSS, and Jinja**. HTML and CSS are used to generate user-facing pages throughout
+  JupyterHub. Jinja is a templating engine that allows you to create HTML pages with
+  variables inserted into them for customization.
+- **Javascript**. Some dynamic content on JupyterHub pages (or via extensions) uses
+  Javascript to control the page's behavior.
+* **UX/UI knowledge**. Aside from specific *implementations* of JupyterHub interfaces
+  (which generally use HTML), it is also helpful to know about the structure and design
+  of user interfaces, and considering the user experience as well.
+
+### Deployment Infrastructure
+
+- **Docker**. JupyterHub uses Docker throughout its projects to provide environments that
+  both JupyterHub and other communities can deploy. Docker is a core part of the Kubernetes
+  deployments of JupyterHub, and can also be used to control user environments in a JupyterHub.
+- **Kubernetes basics**. A popular way to run JupyterHub is by using Kubernetes, a
+  framework for orchestrating cloud infrastructure that is flexible and vendor-agnostic.
+  Some projects of JupyterHub (in particular, [JupyterHub for Kubernetes](https://z2jh.jupyter.org),
+  [Kubespawner](https://jupyterhub-kubespawner.readthedocs.io/en/latest/), and
+  [BinderHub](https://binderhub.readthedocs.io/)) all use Kubernetes for their infrastructure.
+
+### Community infrastructure
+
+* **Technical Writing**. Every tool in the JupyterHub ecosystem is documented, and needs
+  clear, concise, well-structured information to help others learn and use it. Any
+  contributions that improve documentation are appreciated.
+* **Issue guidance and triage**. Almost all enhancements to JupyterHub tools begin with
+  an issue. They are the most common point of interaction with the broader community.
+  To improve the content of our issues and facilitate improvements in JupyterHub, it is
+  important to be a good listener, and to engage with others sharing their perspective.
+
```

Comment:
```suggestion
* **Jupyter Community Forum**. The [Discourse forum](https://discourse.jupyter.org/) is busy with questions and discussions about Jupyter. Support for using JupyterHub is handled through the forum in preference to GitHub issues which are reserved for bug reports and discussion related to technical development. Any help you can offer to other users is very welcome, and the forum is also a great place to exchange knowledge.

```
This paragraph could do with some refinement, but I thought I'd get things started.